src/runtime/runtime_nintendoswitch_heap.go

```diff
@@ -0,0 +1,34 @@
+// +build nintendoswitch
+
+// +build gc.conservative gc.leaking
+
+package runtime
+
+import "unsafe"
+
+const heapSize = 0x2000000 * 16 // Default by libnx
+
+//go:extern _stack_top
+var stackTopSymbol [0]byte
+
+var (
+	heapStart = uintptr(0)
+	heapEnd   = uintptr(0)
+	stackTop  = uintptr(unsafe.Pointer(&stackTopSymbol))
+)
+
+//export setHeapSize
+func setHeapSize(addr *uintptr, length uint64) uint64
+
+//export sleepThread
+func sleepThread(nanos uint64)
```

Comment:
Shouldn't this be in `src/runtime/runtime_nintendoswitch.go`? This does not seem to be associated with the heap.

Reply:
That's true. My bad.